bin/cli.js

```diff
@@ -70,6 +70,22 @@ For more information, see https://webpack.js.org/api/cli/.`);
 		try {
 			options = require("./utils/convert-argv")(argv);
 		} catch (err) {
+			// When webpack is not installed and no args passed to the CLI
+			if (err.code === "MODULE_NOT_FOUND") {
+				let errorMessage =
+					"\n\u001b[31mwebpack not found, please install webpack using\n\t\u001b[33mnpm install --save-dev webpack\n";
+
+				if (process.env.npm_execpath !== undefined) {
```

Comment:
Not really making it more readably by having another if nested, maybe just use `&&` and second condition on newline?

Reply:
@EugeneHlushko  pls review the new changes.